### DISCUSSION TITLE
Complejidad Eventos

### DISCUSSION THREAD

**Discussion by ncruzm** [IIC2133-PUC/2024-2#135] [Tarea 2] [answered]
Hola! Tengo una duda, si no se especifica en el evento por ejemplo en la parte 1.1 de arboles entonces no existe restricción de complejidad?

**Comment by sebacarrascop** [comment, marked as answer]
hola!, se menciona en consideraciones generales:

> No se entregará una complejidad esperada, sin embargo se debe cumplir con los tiempos de ejecución mencionados en la sección de Evaluación.
